src/cache/chrome-storage-assignment-cache.ts

```diff
@@ -0,0 +1,31 @@
+import { AssignmentCacheKey } from '@eppo/js-client-sdk-common/dist/cache/assignment-cache';
+
+import ChromeStorageAsyncMap from './chrome-storage-async-map';
+import { ReadableAssignmentCache } from './hybrid-assignment-cache';
+
+export default class ChromeStorageAssignmentCache implements ReadableAssignmentCache {
+  private readonly storage: ChromeStorageAsyncMap<AssignmentCacheKey>;
+
+  constructor(chromeStorage: chrome.storage.StorageArea) {
+    this.storage = new ChromeStorageAsyncMap(chromeStorage);
+  }
+
+  set(entry: AssignmentCacheKey): void {
+    const { subjectKey, flagKey, allocationKey } = entry;
+    const key = [`subject:${subjectKey}`, `flag:${flagKey}`, `allocation:${allocationKey}`].join(
+      ';',
+    );
+    // "fire-and-forget" - we intentionally don't wait for the promise to resolve
```

Comment:
Is it possible to model this behavior in a spec? I am wondering if this can be called, and then time advanced in jest or with `toHaveBeenCalledWith` ?

Reply:
yeah this is being tested in `HybridStorageAssignmentCache` tests